plugins/module_utils/turbo/module.py

```diff
@@ -80,6 +81,14 @@ def __init__(self, *args, **kwargs):
         )
         self._running = None
         if not self.embedded_in_server:
+            external_libs = [
+                name
+                for name in sys.modules
+                if "site-packages" in str(sys.modules[name])
+                and name not in ["_virtualenv", "distro", "ansible.module_utils.distro"]
```

Comment:
When I tested this locally on the k8s collection I had to add `selinux` here to get it to work.

Reply:
You should not need selinux at this stage.